### DISCUSSION TITLE
Who elects CoCP members?

### DISCUSSION THREAD

**Discussion by kenfinnigan** [commonhaus/foundation#93] [Q&A] [answered]
https://github.com/commonhaus/foundation-draft/blob/main/policies/code-of-conduct.md#code-of-conduct-panel states:

> CoCP members should consider objectivity and diversity of experience and perspective when electing CoCP members.

I believe this should refer to CF members considering objectivity and diversity, as wouldn't it be CF members voting in CoCP elections?

**Comment by ebullient** [comment, marked as answer]
Oh. Yes. Woops.

**Comment by ebullient** [comment]
Resolved by #97
